Tools/publish_multi_request.py

```
@@ -4,26 +4,44 @@
-FORK_ID = "sunrise_station" # Sunrise-Edit - это fallback, настоящее значение передается как аргумент
+# Sunrise edit start - используем единый настраиваемый адрес CDN
+# Параметры инфраструктуры передаются через переменные окружения.
+_robust_cdn_url = os.environ["ROBUST_CDN_URL"].strip()
+_parsed_cdn_url = urlparse(_robust_cdn_url)
+if (
+    _parsed_cdn_url.scheme != "https"
+    or not _parsed_cdn_url.hostname
+    or _parsed_cdn_url.username is not None
+    or _parsed_cdn_url.password is not None
+):
+    raise ValueError("ROBUST_CDN_URL must be an HTTPS URL with a host and without user credentials")
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)